Encode.pm

```diff
@@ -499,30 +499,32 @@ followed by C<encode> as follows:
 
   $octets = encode_utf8($string);
 
+B<WARNING>: L</UTF-8 vs. utf8 vs. UTF8|This function can produce invalid UTF-8!>
```

Comment:
Have you checked output of your change?

I think that this hyperlink does not work.

POD syntax for hyperlinks is `L<text|name/section>` which looks like you swapped _text_ and _link_ parts.

Reply:
I'll fix this one as well